lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -64,39 +64,39 @@ states:
           successful: false
         checkIfDisabled:
           ticf:
-            targetState: COULD_NOT_CONFIRM_DETAILS_PAGE
+            targetState: COULD_NOT_UPDATE_DETAILS_PAGE
```

Comment:
I think this should stay a `COULD_NOT_CONFIRM_DETAILS_PAGE`, as you'll reach here without the feature flag